boards/arm/nrf5340dk_nrf5340/CMakeLists.txt

```diff
@@ -5,4 +5,11 @@ if ((CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP OR CONFIG_BOARD_NRF5340DK_NRF5340_CPU
     AND CONFIG_BOARD_ENABLE_CPUNET)
 zephyr_library()
 zephyr_library_sources(nrf5340_cpunet_reset.c)
+
+if (CONFIG_BUILD_WITH_TFM)
+  zephyr_library_include_directories(
+    $<TARGET_PROPERTY:tfm,TFM_BINARY_DIR>/install/interface/include
```

Comment:
@carlescufi This is now only needed here for the Bluetooth controller debug pins.
Should they also be moved to a GPIO forwarding node?